# CFIBRO.PTML
IFPTML Personalized Medicine Metagenomic Analysis of Microbiome Cystic Fibrosis Patients

#Authors
Carmen Velásquez 1, Estefania Asencio-Medina2,3,6, 
Amaia González-Magaña1, Maria. D Pastor-Viveros1 
Sonia Arrasate3,4, Humberto González-Díaz1,4,5, David Albesa-Jové,1,5

#Affiliations
1 BIOFISIKA: Basque Center for Biophysics CSIC, UPV/EHU), Barrio Sarriena s/n, Leioa, Bizkaia 48940
2 Department of Computer Science and Information Technologies, Faculty of Computer Science, CITIC-Research Center of Information and Communication Technologies, University of A Coruña, Campus Elviña s/n, 15071, A Coruña, Spain.
3 IKERDATA S.L., ZITEK, University of Basque Country UPVEHU, Rectorate Building, 48940 Leioa, Spain.
4Department of Organic and Inorganic Chemistry, Faculty of Science and Technology, University of The Basque Country (UPV/EHU), P.O. Box 644, 48080, Bilbao, Spain.
5IKERBASQUE, Basque Foundation for Science, 48011, Bilbao, Spain.
6Departamento de Bioquímica y Biología Molecular, Universidad del País Vasco UPV/EHU, 48080 Bilbao, España.

# Abstract
It is estimated that in the Basque Country approximately 1 in 3,500 births suffer from Cystic Fibrosis, an autosomal recessive inherited disease, considered the most frequent and severe rare disease in Caucasian, which causes early death (life expectancy estimated at about 50 years). The defective gene is the Cystic Fibrosis Transmembrane Conductance Regulator (CFTR). The main cause of morbidity and mortality is lung disease, due to very thick secretions that obstruct the bronchi, and favor bronchial infection and inflammation. This infection leads to chronic respiratory insufficiency and respiratory failure, resulting in the need for lung transplantation, or even death of the patient. Our working hypothesis and motivations are that improved detection of infections in people with CF, and the development of personalized treatments against these infections will improve and extend the lives of people with CF. To this end, the aim of this work is to perform a targeted metagenomic analysis of CF patient’s microbiome by next generation sequencing (Oxford Nanopore). Here, we show that the microbiome analysis allows to identify pathogens that are not detected by traditional clinical microbiology, provides better quantification, and early diagnosis.  To contribute to improve the life quality and expectancy of CF people, we plan to develop a predictive algorithm based on Artificial Intelligence (AI). This algorithm will merge the microbiome analysis and the clinical data to help clinicians design better treatments. AI offers the opportunity to influence public health, where new capabilities for managing, using and interpreting health data could help better predict and manage disease.

#Data Protection
All data management and analysis were in compliance with the General Data Protection Regulation (GDPR). GDPR defines pseudonymization in Article 4 (5) as: 'the processing of personal data in such a way that they can no longer be attributed to a data subject without the use of additional information, provided that such additional information is kept separately and is subject to technical and organizational measures intended to ensure that the personal data are not attributed to an identified or identifiable natural person'. In general terms, pseudonymization aims at protecting personal data by hiding the identity of individuals (data subjects) in a data set, for example by replacing one or more personal data identifiers with so-called pseudonyms and by adequately protecting the link between the pseudonyms and the initial identifiers. Consequently, all patient data was pseudonymized by the team of the First Affiliated Hospital of Soochow University by using an internal code to identify the patients. These codes and the identities of patients were never shared with the other researchers in this paper. In addition, the pseudonymized data was processed in a single computer and was never copied, transferred, or accessed from/to other devices in the UPV/EHU.

#Supporting Information
Supporting information files contain the following information:

Supporting Information file SI00.doc
Under construction

Supporting Information file SI01.xlsx
Under construction

Supporting Information file SI03.xlsx RON and MON values along with Names and composition of the 10 main components of 5000 fuel blends https://github.com/glezdiazh/BIOFUELS.PTML/blob/main/Supporting%20Information%20file%20SI03.xlsx

Additionally, the data is also released for free in data repository Figshare and indexed with doi number: https://doi.org/10.6084/m9.figshare.21206174.v1.

#ANN code
Supporting information file IFPTML-ANN-CFIBRO.c contain the code in c language of the ANN models trained and validated in this paper. The code of the software used to generate the models is part of the STATISTICA package an is not propietary of the authors for release. The code may be used to implement the model in other software ever following the licence specifications by Statsoft inc. specified inside these files. Please in case you want to obtain similar ANN models use this package or other ANN algorithms package implemented in Phyton, WEKA, or other algorithms. In any case, we would like to point out that the linear models presented in the paper are simpler and have similar to better statistical performance than the ANN models released here. If your objective is using our models to predict new fuel blends we recommend the linear models instead of the ANN models released here. This linear models can be run in Excel or in any other script using the linear equations presented in the paper. You can reproduce the models using the data released at follows in the Supporting information files.

# Funding
Under construction

#References
Under construction


